packages/contracts/contracts/L2/L1Block.sol

```diff
@@ -60,9 +60,10 @@ contract L1Block is Semver {
     uint256 public l1FeeScalar;
 
     /**
-     * @notice The ratio to distribute transaction fees as validator reward. 4 decimal.
+     * @notice The scalar to distribute transaction fees as validator reward.
```

Comment:
To be consistent with the comment above,

```suggestion
     * @notice The scalar value to distribute transaction fees as validator reward.
```

Reply:
Plus, for the same reason, could you specify the denominator of `l1FeeScalar` above?

Reply:
Both applied.